templates/shared/actions/require_action_list.tmpl

```diff
@@ -0,0 +1,145 @@
+							>
+								{{svg "octicon-trash"}}
+							</button>
+							<!-- <a href="{{$.Link}}/{{.ID}}/delete">{{svg "octicon-x-circle-fill"}}</a>-->
+							{{end}}
+						</td>
+					</tr>
+					{{end}}
+				{{else}}
+					<tr>
+						<td class="center aligned" colspan="8">{{ctx.Locale.Tr "actions.require_action.none"}}</td>
+					</tr>
+				{{end}}
+			</tbody>
+		</table>
+	</div>
+	{{template "base/paginate"}}
+</div>
+
+
+
+{{/* Add RequireAction dialog */}}
+<div class="ui small modal" id="add-require-actions-modal">
+	<div class="header">
+		<span id="actions-modal-header">Enable Workflows</span>
+	</div>
+	<form class="ui form form-fetch-action" method="post">
+		<div class="content">
+			<div class="item">
+				<a href="https://docs.gitea.com/usage/actions/require-action">{{ctx.Locale.Tr "actions.require_action.enable_global_workflow"}}</a>
+			</div>
+			<div class="divider"></div>
+			<table class="ui very basic striped table unstackable">
+			<thead>
+				<tr>
+					<th data-sortt-asc="alphabetically" data-sortt-desc="reversealphabetically">
+						{{ctx.Locale.Tr "actions.require_action.workflow"}}
+					</th>
+					<th>
+						{{ctx.Locale.Tr "actions.require_action.repo"}}
+					</th>
+				</tr>
+			</thead>
+			<tbody>
+			{{if .GlobalEnableWorkflows}}
+				{{range .GlobalEnableWorkflows}}
+				<tr>
+				<td><div class="field">
+					<div class="ui radio checkbox">
+						<input class="select-org-radio" name="workflow_name" type="radio" value="{{.Filename}}">
+						<label>{{.Filename}}</label>
+					</div>
+					<input name="repo_name" type="hidden" value="{{.RepoName}}">
+				</div></td>
+				<td><div class="field">
+					<a href="/{{$.OrgName}}/{{.RepoName}}">
+						<label>{{.RepoName}}</label>
+					</a>
+				</div></td>
+				</tr>
+				{{end}}
+			{{else}}
+					<tr>
+						<td class="center aligned" colspan="8">{{ctx.Locale.Tr "actions.require_action.none"}}</td>
+					</tr>
+			{{end}}
+			</tbody>
+			</table>
+			<div class="divider"></div>
+			<div class="item">
+				<a href="{{$.Link}}/add">{{ctx.Locale.Tr "actions.require_action.add_require_action"}}</a>
+			</div>
+		</div>
+		{{template "base/modal_actions_confirm" (dict "ModalButtonTypes" "confirm")}}
+	</form>
+</div>
+<script>
+	window.addEventListener('DOMContentLoaded', function() {
+		var checkboxes = document.querySelectorAll('.ui.radio.checkbox');
+		checkboxes.forEach(function(checkbox) {
+			checkbox.addEventListener('change', function() {
+				var hiddenInput = this.nextElementSibling;
+				var isChecked = this.querySelector('input[type="radio"]').checked;
+				hiddenInput.disabled = !isChecked;
+				// Disable other hidden inputs
+				checkboxes.forEach(function(otherCheckbox) {
+					var otherHiddenInput = otherCheckbox.nextElementSibling;
+					if (otherCheckbox !== checkbox) {
+						otherHiddenInput.disabled = isChecked;
+					}
+				});
+			});
+		});
+	});
+</script>
```

Comment:
This script should be in `web_src/js` and if necessary detect with a fast selector whether the page it's running on is this one.

Reply:
@silverwind does the following make sense? I'm not sure it should be in features or components.  also I am not sure it has a default function to disable input base on select radio button like what I was trying to do.

Reply:
`features` is fine, we pile everything into it currently.

Regarding the other question: I'm not sure but please check how it's done on the repo settings page. I recall there is some mechanism there that enables/disables a bunch of fields when radio buttons are clicked.